### DISCUSSION TITLE
Support for the Fission serverless framework

### DISCUSSION THREAD

**Discussion by facorazza** [redpanda-data/connect#2562] [Ideas]
It'd be nice to have support for [Fission](https://fission.io)

**Comment by mihaitodor** [comment]
Thanks for the suggestion! I have only surface knowledge about Fission. What should that support look like? Note that Benthos is a single static binary that you can run anywhere you like either directly or in a Docker container. Based on [this](https://github.com/fission/examples/tree/main/go/hello-world) Fission hello world example, if you want to craft your own Benthos binary, you can basically copy / paste [`main.go`](https://github.com/benthosdev/benthos-plugin-example/blob/master/main.go) from the [benthos-plugin-example](https://github.com/benthosdev/benthos-plugin-example) repo (without the sample plugins) and write a Benthos yaml config file with a `http_server` input and whichever output.

**Comment by facorazza** [comment]
That's awesome! I haven't had time to test this out but I will soon enough. Perhaps it's just a matter of adding a new page to the docs with the instructions.
Hopefully I'll be able to post an update here

**Comment by mihaitodor** [comment]
Moving this to a discussion as per #2026. Feel free to open a new issue if any Benthos enhancements are required.
